src/traffic_crashlog/backtrace.cc

```diff
@@ -0,0 +1,178 @@
+#include "tscore/ink_config.h"
```

Comment:
We'll need our Apache license preface added to this file.

Reply:
weird that the `RAT` ci didn't fail?

Reply:
it caught it, but didn't fail the build: https://ci.trafficserver.apache.org/job/Github_Builds/job/rat/3830/console